app/javascript/react/components/Button/Button.style.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+import { lightGray } from "../../assets/styles/colors";
+
+const HeaderButton = styled.button`
```

Comment:
For now this is our only button, we know there are more types coming, but we yet need to think about its looks and how to code all the versions. Therefore I would keep the naming consistent and use `Button` here.